### PR TITLE
Fix language direction for RTL support

### LINKED ISSUE #3559: [3.6.0-alpha.2] Language direction broken
**Describe the bug**  

`app` props (`app.$languages`, `app.$system.multilang`, etc..) in direction.js not working but `window.panel.$..` is working.
https://github.com/getkirby/kirby/blob/3.6.0-alpha.2/panel/src/helpers/direction.js

**To Reproduce**  
1. Go to panel
2. Add RTL language
3. Edit any page
4. See error that direction attribute is not appear

**Kirby Version**  

3.6.0-alpha.2

### PR DESCRIPTION
**Pull request by afbora**
## Describe the PR

Fixed language direction attribute for RTL support with switching `window.panel` usage.

### Regression Fixes

- Fixed language direction attribute `#3559`

## Breaking changes

None

## Related issues/ideas

- Fixes #3559 

## Ready?

- [ ] Unit tests for fixed bug/feature
- [x] In-code documentation (wherever needed)
- [x] CI checks pass

## When merging

- [x] Add to [website docs release checklist](https://github.com/getkirby/getkirby.com/pulls) (if needed)
- [ ] Add changes to release notes draft in Notion

### CODE REVIEW

**Comment by afbora** [comment]
I've tested again, actually `app.$languages.current` and `app.$languages.default` doesn't working.

Switched to following:
````js
app.$languages.current -> window.panel.$language
app.$languages.default -> window.panel.$languages.find(language => language.default === true)
````

File: panel/src/components/Forms/Input/SlugInput.vue [hunk 1]
```diff
@@ -53,7 +53,7 @@ export default {
   data() {
     return {
       slug: this.sluggify(this.value),
-      slugs: this.$languages.current ? this.$languages.current.rules : this.$system.slugs,
+      slugs: window.panel.$language ? window.panel.$language.rules : this.$system.slugs,
```
Comment (distantnative) [inline panel/src/components/Forms/Input/SlugInput.vue]: `this.$language` and `this.$language.rules` don't work here?

Comment (afbora) [reply panel/src/components/Forms/Input/SlugInput.vue]: `this.$language` works. When should we choose `this.$..` over `window.panel`? 🙂

**Comment by afbora** [comment]
I've updated PR. Used again `this.$` instead of `window.panel`.

Ready to review 🚀

**Review by distantnative — APPROVED** [review]

**Comment by distantnative** [comment]
While I just approved this... should we actually leave this in JS or instead move it to PHP and pass it via Fiber as `$direction`?

**Comment by afbora** [comment]
So this question is for @bastianallgeier 🙂

**Comment by distantnative** [comment]
Could be this https://github.com/getkirby/kirby/commit/ee3e9482389d17e785bc3ed346f4b1c2dcf3df3f

**Comment by afbora** [comment]
@distantnative Wow! This is great! Lets create PR for it.

**Comment by distantnative** [comment]
Solved via Fiber https://github.com/getkirby/kirby/pull/3568

**Pull request closed without merging**